README.md

```
@@ -1,8 +1,20 @@
-# SNSTR - Simple Nostr Software Toolkit for Retards
+# SNSTR - Secure Nostr Software Toolkit for Renegades
 
-### Still Under Construction 🚧
+![SNSTR Logo](https://raw.githubusercontent.com/AustinKelsay/snstr/main/.github/images/snstr.jpg)
 
-SNSTR is a lightweight TypeScript library for interacting with the Nostr protocol. It provides a simple, easy-to-use API with minimal dependencies.
+### Beta Release 🚧
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)